agent-chat/public/app.css

```
@@ -0,0 +1,228 @@
+  display: inline-flex; align-items: center; gap: 6px; height: 30px; padding: 0 10px;
+  border-radius: 8px; border: 1px solid var(--border); background: transparent;
+  color: var(--text-dim); font: 500 12.5px var(--sans); cursor: pointer; white-space: nowrap;
+  max-width: 260px; user-select: none;
+}
+.chip:hover { background: var(--bg-hover); color: var(--text); }
+.chip:disabled, .chip[data-disabled] { opacity: .48; cursor: default; }
+.chip .chev { display: inline-flex; opacity: .7; margin-left: 1px; }
+.select-trigger[data-popup-open] .chev { transform: rotate(180deg); }
+.cwd-label, .select-value { overflow: hidden; text-overflow: ellipsis; }
+.cwd-label { font-family: var(--mono); font-size: 11.5px; }
+.dot { width: 9px; height: 9px; border-radius: 50%; flex: none; background: var(--text-faint); box-shadow: 0 0 0 2px color-mix(in srgb, currentColor 12%, transparent); }
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)